src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -174,10 +174,22 @@ func (w logglyErrorWriter) Write(b []byte) (int, error) {
 		"timeZone":  w.tz,
 		"version":   w.versionToLoggly,
 	}
-
+	fullMessage := string(b)
+	message := fullMessage
+	// Loggly can't index fields with more than 100 characters
```

Comment:
Small nitpick, but I think Loggly can still index these fields, it just doesn't generated facets for them.